plugins/core/jetbrains-community/src/icons/AwsIcons.kt

```diff
@@ -145,6 +145,30 @@ object AwsIcons {
         @JvmField val CHECKMARK_GRAY = load("/icons/resources/codetransform/checkmark.svg")
 
         @JvmField val TIMELINE_STEP = load("/icons/resources/codetransform/transform-timeline-step-done.svg") // 16 * 16
+
+        @JvmField val PLAN_VARIABLES_LIGHT = load("/icons/resources/codetransform/transform-variables-light.svg")
+
+        @JvmField val PLAN_VARIABLES_DARK = load("/icons/resources/codetransform/transform-variables-dark.svg")
```

Comment:
`AwsIcons` and `MessagesBundle` are explicitly marked as deprecated, but i understand the time constraints